### PR TITLE
PRIME 206 Upload Documents on Enrollee

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-angular-frontend/src/app/shared/components/document-upload/document-upload/document-upload.component.ts [hunk 1]
```diff
@@ -73,6 +71,10 @@ export class DocumentUploadComponent implements OnInit {
     };
   }
 
+  public removeFiles() {
+    this.filePondComponent.removeFiles({ revert: false });
```
Comment (AnaisHebert) [inline prime-angular-frontend/src/app/shared/components/document-upload/document-upload/document-upload.component.ts:75]: image

**Comment by mtpultz** [comment]
@hogmann I pushed a temporary fix for the typing error

File: prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-notes/enrollee-adjudicator-notes.component.spec.ts~HEAD [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-notes/enrollee-adjudicator-notes.component.spec.ts~HEAD:1]: Looks like you had an issue while merging

File: prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-notes/enrollee-adjudicator-notes.component.spec.ts~develop [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-adjudicator-notes/enrollee-adjudicator-notes.component.spec.ts~develop:1]: Looks like you had an issue while merging

File: prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AdjudicatorDocumentsComponent } from './adjudicator-documents.component';
+
+describe('AdjudicatorDocumentsComponent', () => {
+  let component: AdjudicatorDocumentsComponent;
+  let fixture: ComponentFixture<AdjudicatorDocumentsComponent>;
+
+  beforeEach(async(() => {
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.spec.ts:9]: Can the tests still be run?

**Review by AnaisHebert — APPROVED** [review]
🧇🥞

**Pull request merged**